types/index.d.ts

```diff
@@ -53,6 +53,7 @@ interface Wrapper {
     update(): void;
     setData(data: object): void;
     setProps(props: object): void;
+    setMethods(methods: object): void;
```

Comment:
Can you add a test for this in this file - https://github.com/eddyerburgh/avoriaz/blob/master/types/test/mount.spec.ts#L72